src/OrchardCore.Modules/OrchardCore.Users/Controllers/AccountController.cs

```diff
@@ -760,60 +732,34 @@ public async Task<IActionResult> RemoveLogin(RemoveLoginViewModel model)
         private async Task<string> GenerateUsername(ExternalLoginInfo info)
         {
             var now = new TimeSpan(_clock.UtcNow.Ticks) - new TimeSpan(DateTime.UnixEpoch.Ticks);
-            var ret = string.Concat("u" + Convert.ToInt32(now.TotalSeconds).ToString());
+            var ret = String.Concat("u" + Convert.ToInt32(now.TotalSeconds).ToString());
```

Comment:
so is this the default user name it generates?

How can total seconds be unique? Wouldn't it be safer for this to be a guid, but onlye generated if there was no result?

Reply:
I was thinking about that too, felt like there could easily be name collisions here on a high traffic website.

Reply:
@deanmarcussen do you agree on the change to a uuid?